### PR TITLE
Add draft orders query

### LINKED ISSUE #3796: Add query to fetch draft orders
In the Dashboard 2.0 we want to have separate links to view draft orders and confirmed ones. For that reason we need to add a separate resolver to return only draft orders. Exiting `orders` resolver should be changed to return confirmed orders.

### PR DESCRIPTION
**Pull request by michaljelonek**
Adds draft orders query.

Resolves #3796.

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] Database migration files are up to date.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: saleor/graphql/order/schema.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     OrderUpdate, OrderUpdateShipping, OrderVoid)
 from .resolvers import (
     resolve_homepage_events, resolve_order, resolve_order_by_token,
-    resolve_orders, resolve_orders_total)
+    resolve_orders, resolve_orders_total, resolve_draft_orders)
```
Comment (jxltom) [inline saleor/graphql/order/schema.py]: Don't forget to sort the imports by ```isort```:D

Comment (michaljelonek) [reply saleor/graphql/order/schema.py]: lakhgoklwbeglksdng 😢 

I read need to get those git hooks going, I always forget.

File: saleor/graphql/schema.graphql [hunk 2]
```diff
@@ -1254,8 +1254,10 @@ enum OrderStatus {
 }
 
 enum OrderStatusFilter {
-  READY_TO_FULFILL
```
Comment (jxltom) [inline saleor/graphql/schema.graphql]: So we can't filter by READY_TO_FULFILL and READY_TO_CAPTURE, right? But i think storefont 2.0 is using them?

Comment (michaljelonek) [reply saleor/graphql/schema.graphql]: Ah, you're right. It probably does use them. I'll add them back for backward compatibility reasons, but sadly I can't mark that in the API.

Comment (michaljelonek) [reply saleor/graphql/schema.graphql]: Or actually, I'll revert that change and we can come back to it at a later stage. Nothing uses it anyway.

**Comment by auvipy** [comment]
tests seem perfect.

**Comment by maarcingebala** [comment]
After merging this Dashboard 2.0 won't show draft orders in "Orders" section. This will be fixed in PR #3856.

**Pull request merged**